sysroot_creator/scripts/create_cc_sysroot

```diff
@@ -277,33 +320,50 @@ def main():
         args.docker_network_mode,
         args.sysroot_nocache)
 
+    # Main pipeline
+    pbar = tqdm(total=9, unit='step')
+    step = 1
+    pbar.update(step)
+
     cc_root_dir = setup_cc_root_dir(platform)
+    pbar, step = update_pbar(pbar, step)
+
     sysroot_dir = setup_sysroot_dir(cc_root_dir, args.force_sysroot_build)
+    pbar, step = update_pbar(pbar, step)
+
     docker_image_tag = get_workspace_image_tag(platform)
+    pbar, step = update_pbar(pbar, step)
+
     build_workspace_sysroot_image(platform, docker_args, docker_image_tag)
+    pbar, step = update_pbar(pbar, step)
+
     export_workspace_sysroot_image(docker_image_tag, sysroot_dir)
+    pbar, step = update_pbar(pbar, step)
 
     cc_build_setup_file_path = write_cc_build_setup_file(platform, cc_root_dir)
+    pbar, step = update_pbar(pbar, step)
+
     # generalization of the Poco hack
     # from https://github.com/ros2/cross_compile/blob/master/entry_point.sh#L38
-    cc_system_setup_script_path = write_cc_system_setup_script(
-        cc_root_dir, platform)
-
-    print("""
-
-    In order to complete the cross compilation setup, please
-
-    1. WORKAROUND: run the command below to setup using sysroot's GLIBC for
-    cross compilation.
-
+    cc_system_setup_script_path = write_cc_system_setup_script(cc_root_dir, platform)
+    pbar, step = update_pbar(pbar, step)
+
+    setup_sysroot_environment(cc_system_setup_script_path, cc_build_setup_file_path)
+    pbar, step = update_pbar(pbar, step)
+    pbar.close()
+
+    logger.warning("""
```

Comment:
This can be removed now, since it is done from the script itself now?

Reply:
Left it so users could know in the future how to setup their environment. I think it might be better to combine them into one script tho.